### ISSUE TITLE
Fatal Error - Lists are not supported by getDocument() yet

### ISSUE THREAD

**Issue by danjaywing** [lanthaler/JsonLD#75]
Hi, I encountered the following error whilst using 'jkphl/micrometa', which uses JsonLD:

'Fatal Error - Lists are not supported by getDocument() yet'

Here is where the issue was initially raised: https://github.com/jkphl/micrometa/issues/12

Does the 'yet' imply that lists are to be supported at some stage?

**Comment by lanthaler** [comment]
Yes, but probably not anytime soon as I don't have the necessary time. Shouldn't be too difficult to implement though if you want to give it a shot.
